ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java

```diff
@@ -42,6 +42,13 @@ public class DssDatabase implements EnsembleDatabase,MetricDatabase {
     static String metricTimeseriesIdentifier = "MetricTimeseries";
     static String metricPairedDataIdentifier = "MetricPairedData";
     boolean CatalogIsUpToDate = false;
+    private String overriddenFPart = null;
+
+    public void setOverriddenFPart(String overriddenFPart) {
```

Comment:
For this override F part  I have two thoughts to share:
1) consider calling this user-supplied f-part (from the perspective of DssDatabase clients)
2) consider a callback or event or inheritance that actually does the override outside the DssDatabase.